### DISCUSSION TITLE
Finetuning MACE models

### DISCUSSION THREAD

**Discussion by lucasfrguez** [ACEsuit/mace#604] [General]
Hello, MACE community.

I was checking the recently uploaded MACE tutorials. on Google Collab and everything worked smoothly (I ran them locally) except for the last part when I tested the different multi-head options, it seems I simply ran out of memory; is there a way to circumvent this? Since the training set is not really big, I do not see how I can divide them into chunks of .h5 files.

I already saw that this is still a beta function and I guess further info will be provided but I wanted to make sure, the multi-head replay right now is not available for mace-off models, is it correct?

Thanks you in advance
Lucas

**Comment by ilyes319** [comment]
Hello,

To reduce the memory, you can try to reduce the number of configurations that are kept for the replay,, by changing `--num_samples_pt` to something like 50. Are you running on GPUs?

It is supported but not yet documented. It will come soon hopefully.

**Reply by lucasfrguez** [reply]
Yes, I am using 1 GPU with ~ 8 GB of memory, indeed, reducing the number of samples worked fine, thank you!
